modules/apps/object/object-api/src/main/java/com/liferay/object/field/business/type/attachment/AttachmentUploadHelper.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.field.business.type.attachment;
+
+import com.liferay.object.field.render.ObjectFieldRenderingContext;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.repository.model.Folder;
+
+/**
+ * @author Carolina Barbosa
+ */
+public interface AttachmentUploadHelper {
```

Comment:
Actually, it was because I was getting inspiration from `DDMFormUpload...`. But now that you mention it, I think just `AttachmentHelper` might be enough. Or even `AttachmentObjectFieldBusinessTypeHelper`. What do you think?

Reply:
Sounds good. 😄